src/contexts/AuthUIContextProvider/types.ts

```diff
@@ -142,6 +142,13 @@ type AuthUIContextProviderProps = {
      * Default: None
      */
     customAccountDetails?: Array<CustomRegistrationForm | null>;
+
+    /**
+     * Custom screen to render when a user successfully completes registration.
+     *
+     * Default: Provides a generic success screen with Name, Email, and Org details.
+     */
+    customRegistrationSuccessScreen?: JSX.Element;
```

Comment:
You're probably going to want to do this like we do for the custom header and footer for the login page and let it be a JSX Element or a function that we can pass the navigation object to (for cross-compatibility for react/RN):

```loginHeader?: JSX.Element | ((navigation: any) => JSX.Element);```

That will give us access to the navigation prop in RN too so the user can handle the navigation back to the login screen or wherever they want to go.

Reply:
otherwise looks good so far